README.md

```diff
@@ -4,7 +4,7 @@
 
 This really easy to use code developed since 2016 analyses research outcomes provided by the EPSRC.
 
-The results of the investigation is published on the Software Sustainability Institute's website under the title: "[Researchfish®: what can it tell us about software in research?](https://www.software.ac.uk/blog/typo/2017-01-18-researchfishr-what-can-it-tell-us-about-software-research)".
+The results of the investigation is published on the Software Sustainability Institute's website under the title: "[Researchfish®: what can it tell us about software in research?](https://www.software.ac.uk/blog/2017-01-18-researchfishr-what-can-it-tell-us-about-software-research)".
```

Comment:
Why not also add a link to the zenodo content? It would be `https://doi.org/10.5281/zenodo.250494`